.github/workflows/release.yml

```diff
@@ -0,0 +1,83 @@
+name: Release
+
+on:
+  schedule:
+    - cron: '0 4 * * 1-5'
+
+jobs:
+  release:
+    name: 'Release & Publish'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v2
+        with:
+        # pulls all commits (needed for lerna / semantic release to correctly version)
+         fetch-depth: 0
+         token: ${{ secrets.GH_TOKEN }}
+
+      - name:  Pull all tags
+        # pulls all tags (needed for lerna / semantic release to correctly version)
+        run: git fetch --depth=1 origin +refs/tags/*:refs/tags/*
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: 13
+          registry-url: 'https://registry.npmjs.org'
+
+      - name: Install dependencies
+        run: yarn install
+
+      - name: Get version from lerna.json before release step
+        id: initversion
+        run: |
+          echo "::set-output name=version::$(grep '"version":' lerna.json | cut -d\" -f4)"
+
+      - name: Check for package changes
+        id: changes
+        run: |
+          npx lerna changed  2>&1 | tail -1 |tee outfile
```

Comment:
npx downloads and then runs Lerna - not sure if there is some overhead here? 

Maybe that's not an issue and the package gets cached.

Reply:
I found that within the runner we get 'lerna not found' so using npx to ensure it will always run, overhead should be minimal.